fuzz-tests/Cargo.toml

```diff
@@ -9,26 +9,38 @@ cargo-fuzz = true
 
 [dependencies]
 libfuzzer-sys = { version = "0.4", optional = true }
-afl = { version = "0.12.16", optional = true }
+afl = { version = "0.12.16", features = ["reset_lazy_static"], optional = true }
 clap = { version = "4.1.7", optional = true }
 rand = { version = "0.8.5", optional = true }
 rand_chacha = { version = "0.3.1", optional = true }
 log = { version = "0.4.17", optional = true }
 once_cell = { version = "1.17.1"}
 sbor = { path = "../sbor", default-features = false }
-radix-engine = { path = "../radix-engine", default-features = false }
+radix-engine = { path = "../radix-engine", default-features = false, features = ["radix_engine_fuzzing"] }
 radix-engine-constants = { path = "../radix-engine-constants" }
 radix-engine-interface = { path = "../radix-engine-interface", default-features = false }
+radix-engine-stores = { path = "../radix-engine-stores", default-features = false }
 transaction = { path = "../transaction", default-features = false }
-utils = { path = "../utils", default-features = false }
+utils = { path = "../utils", default-features = false, features = ["radix_engine_fuzzing"] }
 scrypto-unit = { path = "../scrypto-unit", default-features = false }
 
+[patch.crates-io]
+# Use "resettable" `lazy_static` version to prevent incorrect AFL behaviour, eg.
+# - warnings such as "Instrumentation output varies across runs"
+# - low stability reports
+# More details: https://github.com/rust-fuzz/afl.rs#lazy_static-variables
+lazy_static = { git = "https://github.com/rust-fuzz/resettable-lazy-static.rs" }
```

Comment:
If we do this we need to fix the commit hash to prevent injection risks.

Reply:
Good point! Thanks @dhedey

Reply:
Is it possible to specify the full git hash? A collision of 7 hex characters is easily brute forcible.

Reply:
Sorry, missed your comment and merged 😕 
Separate PR created for this: https://github.com/radixdlt/radixdlt-scrypto/pull/971